match/lib/match/nuke.rb

```diff
@@ -27,15 +27,15 @@ def run(params, type: nil)
       end
 
       if (self.certs + self.profiles + self.files).count > 0
-        unless params[:force]
+        unless params[:unattended]
```

Comment:
Documentation seems to indicate that both `-unattended` and `-force` are required, but it looks like only checking `params[:unattended]` here.